dom.go

```diff
@@ -3026,6 +3028,12 @@ type HTMLUnknownElement struct{ *BasicHTMLElement }
 
 type HTMLVideoElement struct{ *HTMLMediaElement }
 
+type HTMLTemplateElement struct{ *BasicHTMLElement }
+
+func (e *HTMLTemplateElement) Content() DocumentFragment {
+	return wrapDocumentFragment(e.Get("content"))
```

Comment:
Question, do you know if `e.Get("content")` always returns a non-nil value?

Reply:
I see, thanks for clarifying. Then let's keep this as is. If we discover evidence that there are reproducible cases where it's nil and needs to be handled, we can address it then.